src/modules/Elsa.Hosting.Management/Notifications/InstanceDeactivated.cs

```diff
@@ -0,0 +1,5 @@
+using Elsa.Mediator.Contracts;
+
+namespace Elsa.Hosting.Management.Notifications;
+
+public record InstanceDeactivated(string InstanceName) : INotification;
```

Comment:
As a matter of personal preference, I think that `HeartbeatTimedOut` would has a better ring to it, and still makes sense since this event means that a given instance failed to send a heartbeat (i.e. it exceeded a certain time out threshold).